doc/CMakeLists.txt

```diff
@@ -47,6 +47,12 @@ set(SPHINX_ENV
   DOT_EXECUTABLE=${DOXYGEN_DOT_EXECUTABLE}
 )
 
+if(EXISTS "${ZEPHYR_BASE}/doc/custom.doxyfile")
```

Comment:
Would it be better to have this custom file outside of zephyr source tree, perhaps pointed by env var? That way user would not need to clutter git tree with out of tree files.

Reply:
Changed to use an environment variable.